clientupdate/clientupdate.go

```diff
@@ -99,12 +102,19 @@ func (args Arguments) validate() error {
 	if args.Logf == nil {
 		return errors.New("missing Logf callback in Arguments")
 	}
+	if args.Version != "" && args.Track != "" {
+		return errors.New("only one of Version or Track can be set")
+	}
+	switch args.Track {
+	case StableTrack, UnstableTrack, CurrentTrack:
```

Comment:
(optional) When I'm intentionally leaving a switch case empty, I like to leave a comment for the reader. (Even though Go switches don't fall through, a lot of folks are accustomed to C, Java, etc.)
```suggestion
	case StableTrack, UnstableTrack, CurrentTrack:
		// OK
```